### PR TITLE
Add wheel package to all containers

### PR DESCRIPTION
**Pull request by asmorodskyi**

### CODE REVIEW

**Review by ricardobranco777 — APPROVED** [review]
LGTM

File: ocw/lib/gce.py [hunk 1]
```diff
@@ -59,7 +60,19 @@ def _delete_resource(self, api_call, resource_name, *_, **kwargs) -> None:
...
+                # These are system generated routes when you create a network. These
+                # will be deleted by the deletion of the network and do not block the
+                # deletion of that network.
+                # There are no properties on the Route struct that indicate a route is a
+                # default one. Typically, the name will contain the word "default" or the
+                # description will contain the word "Default" but a property like Kind
+                # returns "compute#route" for all routes.
+                # All this creating false alarms in log which we want to prevent.
+                # Only way to prevent is mute error
+                if resource_type.title() == "Route" and "The local route cannot be deleted" in str(err):
```
Comment (ricardobranco777) [inline ocw/lib/gce.py:72]: Why use `.title()` here?

**Review by ricardobranco777 — APPROVED** [review]
Great refactoring!

I will take a look at https://pypi.org/project/requests-mock/ on a side project and then perhaps reuse here.

LGTM

File: ocw/lib/gce.py (hunk 1 above)
Comment (asmorodskyi) [reply ocw/lib/gce.py:72]: I don't know I was just copy/paste this from your code above :) returning question back to you :)

Comment (ricardobranco777) [reply ocw/lib/gce.py:72]: I don't remember XD

**Pull request merged**